src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -7099,9 +7098,12 @@ public static void Main(string[] args)
     }
 }
 ";
+            CreateCompilation(source, parseOptions: TestOptions.Regular11).VerifyDiagnostics(
+                // (5,27): error CS8652: The feature 'lambda optional parameters' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                //         var lam1 = (int x = 7) => x;
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "=").WithArguments("lambda optional parameters").WithLocation(5, 27));
 
-            var comp = CreateCompilation(source);
-            comp.VerifyDiagnostics();
+            CreateCompilation(source, parseOptions: TestOptions.RegularNext).VerifyDiagnostics();
```

Comment:
nit: Consider adding LangVer variants for the `delegate (params ...)` and `delegate (... = defaultValue)` scenarios as well.

Reply:
Even though there is no change between lang versions? (I.e., `delegate { }` syntax wasn't and won't be supported.)

Reply:
Yes, the reason I'm suggesting those tests is that the new diagnostic is conditional on `isAnonymousMethod` and `parsingLambdaParams` (vs. `parsingAnonymousMethodParams`). 
The code shows that the LangVer checks will not apply to `delegate (...)` syntax, so that's why I'd consider capturing that behavior.

Reply:
Thanks, added: f128a01429500794dd0747055c1a9384233206e6

Although one thing is not clear to me. Previously, I used `TestOptions.Regular11` and `TestOptions.RegularNext`. Now I wanted to use `Theory` but then I have to use `LanguageVersion` and there seems to be no equivalent to `TestOptions.RegularNext`, there's only `LanguageVersion.Preview`, is that correct?

Reply:
For `LanguageVersion` you can use `LanguageVersionExtensions.CSharpNext`. `LanguageVersion` is public so we don't want an entry called "Next" in that list (that's purely an internal concern to help our work/release).